API/Tests/BaseTest.cs

```diff
@@ -12,15 +12,12 @@ namespace API.Tests
     public class BaseTest
     {
         protected APIUtils.API api;
-        protected Client client;
         protected ICustomLogger logger;
-
         protected static ApiResourcesModel ApiResourcesData;
         
         [SetUp]
         public void SetUp()
         {
-            client = Client.Instance;
```

Comment:
Client.Instance must be initialized at least once. API.cs using this instance to get the response. It's not really clear but this line should be here or you may add null check for Client.GetClient()

Reply:
Ok noted Yauheni Thanx